apps/spruce/src/pages/adminSettings/Tabs.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import useScrollToAnchor from "hooks/useScrollToAnchor";
+import { GeneralTab } from "./tabs/GeneralTab/GeneralTab";
+
+export const AdminSettingsTabs = () => {
+  useScrollToAnchor();
```

Comment:
The implementation for useScrollToAnchor may have to change a little if we go with the format `/admin-settings/jira` since the code for useScrollToAnchor looks for a `#`